packages/ui-components/src/components/Message/Message.component.tsx

```diff
@@ -0,0 +1,259 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState, useRef } from "react"
+
+import { Icon } from "../Icon"
+import { KnownIcons, KnownIconsEnum } from "../Icon/Icon.component"
+
+const messageBaseStyles = `
+    jn-text-theme-high
+    jn-flex
+    jn-rounded
+    jn-leading-5
+    jn-overflow-hidden
+    jn-items-center
+`
+
+const messageHeadingStyles = `
+    jn-font-bold
+`
+
+const messageContentStyles = `
+    jn-py-3
+    jn-pr-4
+    jn-ml-7
+`
+
+const messageBorderStyles = `
+    jn-w-[4px]
+    jn-self-stretch
+    jn-border-l-4
+    jn-mr-6
+    jn-shrink-0
+`
+
+const dismissButtonStyles = `
+    jn-ml-auto
+    jn-self-stretch
+    jn-flex
+    jn-flex-col
+    jn-py-2.5
+    jn-pr-2.5
+`
+
+const messageVariantStyles = {
+  default: `jn-border-theme-message-default`,
+  defaultBg: `jn-bg-theme-message-default`,
+
+  error: `jn-border-theme-message-error`,
+  errorBg: `jn-bg-theme-message-error`,
+
+  warning: `jn-border-theme-message-warning`,
+  warningBg: `jn-bg-theme-message-warning`,
+
+  danger: `jn-border-theme-message-danger`,
+  dangerBg: `jn-bg-theme-message-danger`,
+
+  success: `jn-border-theme-message-success`,
+  successBg: `jn-bg-theme-message-success`,
+}
+
+export type VariantType = "info" | "warning" | "danger" | "error" | "success"
```

Comment:
I would like to rename it here same way as we did with badge

Reply:
Ah, good catch!